src/js/actions/auth-actions.js

```diff
@@ -0,0 +1,15 @@
+export const SET_AUTH = 'SET_AUTH';
+export const RESET_AUTH = 'RESET_AUTH';
+
+export function setAuth(props) {
+  return {
+    type: SET_AUTH,
+    props: props
+  };
+}
+
+export function resetAuth() {
```

Comment:
To this question, I'd say : check the reducer.

In short, it removes all traces of jwt or app token from the store.